src/zimscraperlib/zim/creator.py

```diff
@@ -146,7 +150,31 @@ def config_indexing(
         self.__indexing_configured = True
         return self
 
+    def _maybe_metadata_from_maybe_illustration(self, name, value):
+        if not isinstance(name, str) or not name.startswith("Illustration_"):
+            return None
+        if isinstance(value, bytes):
+            image = io.BytesIO(value)
+        try:
+            img = PIL.Image.open(image)
```

Comment:
Any reason to re-implement this rather than using https://github.com/openzim/python-scraperlib/blob/cc0cceb6a1c4afd2b810db841155c9ec5ffad2ed/src/zimscraperlib/image/probing.py#L54 ?